File: backend/src/main/java/com/denyyyys/fluentLift/model/postgres/dto/response/AppUserResponseDto.java
package com.denyyyys.fluentLift.model.postgres.dto.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class AppUserResponseDto {
    private String name;
    private String email;
    private Long id;
}

File: mobile/app/src/main/java/com/example/navigation/data/dto/response/AppUserResponseDto.kt
package com.example.navigation.data.dto.response

import com.google.gson.annotations.SerializedName

data class AppUserResponseDto(
    @SerializedName("name") val name: String,
    @SerializedName("email") val email: String,
    @SerializedName("id") val id: Long
)